products/aws-lambda.md

```diff
@@ -199,13 +199,13 @@ releases:
 
 -   releaseCycle: "nodejs12.x"
     releaseLabel: Node.js 12
-    releaseDate: 2021-02-03
+    releaseDate: 2019-11-18
```

Comment:
In preview for this entry still shows: 03 Feb 2021
Is this due to automation?

Reply:
Yes, probably. aws-lambda's script is a bit special, and is using product's releaseDate as the source for the date. So merging this should fix the issue (after the next release-data update).

Reply:
Ok then I merge this now and will check...

Reply:
Worked :+1: